src/service/fixed-layer.js

```diff
@@ -449,19 +452,40 @@ export class FixedLayer {
           // We shouldn't have too many of `fixed` elements.
           break;
         }
-        this.setupElement_(elements[j], fixedSelector, 'fixed');
+        const el = elements[j];
+        if (isInlineStylesEnabled) {
+          this.sanitizeElement_(el);
+        }
+        this.setupElement_(el, fixedSelector, 'fixed');
       }
     }
     for (let i = 0; i < stickySelectors.length; i++) {
       const stickySelector = stickySelectors[i];
       const elements = this.ampdoc.getRootNode().querySelectorAll(
           stickySelector);
       for (let j = 0; j < elements.length; j++) {
-        this.setupElement_(elements[j], stickySelector, 'sticky');
+        const element = elements[j];
+        if (isInlineStylesEnabled) {
+          this.sanitizeElement(element);
+        }
+        this.setupElement_(element, stickySelector, 'sticky');
       }
     }
   }
 
+  /**
+   * Sanitizes an element, cleaning up the style field attribute.
+   * @param {!Element} element
+   * @private
+   */
+  sanitizeElement_(element) {
+    if (element.style) {
```

Comment:
This is always true. Use `hasAttribute`.

Reply:
done